tools/rna_tools/cmsearch_deoverlap/cmsearch-deoverlap.xml

```diff
@@ -16,17 +17,17 @@
         #end if
         #if $dirty:
             $dirty &&
-            sed 's/[:blank:]+/,/g' cmsearch_matches.sort > cmsearch_matches.sort.tabular &&
+            sed "s/[[:blank:]]\+/\t/g" cmsearch_matches.sort > cmsearch_matches.sort.tabular &&
             mv cmsearch_matches.sort.tabular '$matches_sort'
         #end if
         &&
-        sed 's/[:blank:]+/,/g' cmsearch_matches.deoverlapped > cmsearch_matches.deoverlapped.tabular &&
+        sed "s/[[:blank:]]\+/\t/g" cmsearch_matches.deoverlapped > cmsearch_matches.deoverlapped.tabular &&
         mv cmsearch_matches.deoverlapped.tabular '$deoverlapped_matches'
 
     ]]></command>
     <inputs>
         <param name="clan_information" type="data" format="txt" label="Clan information" help="Each row in the file contains an RNA clan and its corresponding family members"/>
-        <param name="cmsearch_matches" type="data" format="tabular" label="The tabular result file from cmsearch"/>
+        <param name="cmsearch_matches" type="data" format="txt" label="The result file from cmsearch"/>
```

Comment:
why is the result file txt and not tabular?

Reply:
> why is the result file txt and not tabular?

The result you get from cmsearch on galaxy is a tabular (is formatted to tabular).
Yet i thought it would be helpful to change to txt, in case some users want to use the tool with cmsearch results that were not generated on galaxy.